src/util/ConfigManager/ConfigManager.cpp

```diff
@@ -690,12 +765,71 @@ void ConfigManager::verifyWithValidators() const {
 
 // ____________________________________________________________________________
 bool ConfigManager::containsOption(const ConfigOption& opt) const {
-  const auto allOptions = configurationOptions(false);
+  const auto allOptions = configurationOptions();
   return ad_utility::contains(
       std::views::values(allOptions) |
           std::views::transform(
               [](const ConfigOption& option) { return &option; }),
       &opt);
 }
 
+// ____________________________________________________________________________
+template <typename T>
+requires std::same_as<T, ConfigOption> || std::same_as<T, ConfigManager>
+void ConfigManager::ConfigurationDocValidatorAssignment::addEntryUnderKey(
+    const T& key, const ConfigOptionValidatorManager& manager) {
+  // The concerned hash map.
+  MemoryAdressHashMap<T>& hashMap{getHashMapBasedOnType<T>()};
+
+  // If we already have a vector, append. Else, create.
+  if (auto mapIterator = hashMap.find(&key); mapIterator != hashMap.end()) {
+    (*mapIterator).second.push_back(&manager);
+  } else {
+    hashMap.emplace(&key, std::vector{&manager});
+  }
+  validatorSet_.emplace(&manager);
```

Comment:
We think the `validatorSet_` is not needed anymore, throw it out.

Reply:
Done, see [38924f8](https://github.com/ad-freiburg/qlever/pull/1169/commits/38924f8e98f21e7b0037f65115f14db4f235d8c9).